src/audio/native.cmake

```diff
@@ -0,0 +1,108 @@
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
"native" could be a bit ambiguous, lets spell out plugin.cmake and have a comment here that says this is to build plugin, testbench etc.

Reply:
We had the same discussion when coining CONFIG_COMP_MODULE_SHARED_LIBRARY_BUILD . Maybe stick to that name and have "audio/module_shared_library.cmake". I can live with "plugin.cmake" as well.